bitrise.yml

```diff
@@ -237,3 +247,14 @@ workflows:
             set -e
             make dogfood ci=${CI}
 
+  ship:
+    description: |-
+        Ships the tagged version to Cocoapods trunk and uploads binaries to Github.
+    steps:
+    - script:
+        title: Ship tagged version
+        inputs:
+        - content: |-
+            #!/usr/bin/env zsh
+            set -e
+            make ship
```

Comment:
Once you mentioned making `DatadogSDKObjc/AlamofireExtension` subspecs of `DatadogSDK` so that only one `podspec` will need to be pushed, i think that makes the most sense in this case as it solves the root cause.

we can arrange `DatadogSDK.podspec` in the following way:
```
s.default_subspec = 'Swift`

s.subspec 'Swift' do ... end

s.subspec 'Objc' do ... end
s.subspec 'AlamofireExtension' do ... end
```
this shouldn't be a breaking change for Objc users.
also, in the next `DatadogSDKObjc` version we can add warnings to the console to direct users to use `pod DatadogSDK/Objc` instead of `pod DatadogSDKObjc`.

does it make sense until here?
if so, i will verify that this won't be a breaking change for Swift users neither (they may need to import `DatadogSDKSwift` in their source code but i haven't verified yet)

Reply:
Sure, our `.podspec` layout is not ideal and it forces us to submit 3 specs, instead of one. This is where the issue originates - submitting 2nd and 3rd podspec requires waiting until CP processes the 1st one. It would be ideal to fix the root cause, agree.

However, changing to `subspecs` model will require refactoring in numerous Objc SDK files and taking additional decision for `AlamofireExtension` layout. Note that with CP subspecs the codedbase is not modeled as modules, but rather a compilation units with different sets of source files. The `import Datadog` will not work neither in `Objc` spec, nor in `AlamofireExtension`. We would need to make each import conditional with `#if !COCOAPODS` to make it work. In result, the modules layout for CP will be different than the one for SPM and Carthage.